internal/tiltfile/encoding/json_test.go

```diff
@@ -118,3 +168,27 @@ test()
 	}
 	require.NoError(t, err)
 }
+
+func TestEncodeJSONNonStringMapKey(t *testing.T) {
```

Comment:
Is there a typo here and in the test below?

```suggestion
func TestEncodeJSONOnStringMapKey(t *testing.T) {
```

Reply:
nope! JSON only supports strings as map keys, and this test is for serializing a map that has a map key that's not a string and making sure returns an error that map keys must be strings.